cypress/e2e/visit-homepage.cy.js

```diff
@@ -0,0 +1,13 @@
+describe('Visit homepage', () => {
+  it('visit homepage', 
+  () => {
+    if(Cypress.env('actual_env')=="qa"){
```

Comment:
This if/else is unnecessary here and everything can work without it.

It can be simplified further by using dotenv to use baseUrl property.